### ISSUE TITLE
/dev/null -- Improve accuracy.

### ISSUE THREAD

**Issue by Altorffer** [OpenMods/OpenBlocks#580]
/dev/null is a pretty nifty little gimmick, however, it does have a few issues. Primarily, I suspect whoever implemented it had a very specific need in mind, limiting it merely to simply grabbing superflous dirt and cobblestone while mining. Because that is essentially what it is good for right now, grabbing things as your inventory would slowly fill up by picking those items from the ground or such.

What I would like to see is for /dev/null to be a more accurate grabber; for example, it will NOT trigger when collecting thousands of items from, say, an Aethercraft Manipulator. Or any inventory, I suspect. Additionally, it does not trigger for items already present in your inventory. For example, I have a /dev/null with a dirt in my backpack, just to be able to throw dirt its way whenever and wherever. Now, I forget to take it out and start digging, filling my inventory with needless dirt. I remember my /dev/null and take it out of the backpack. It will NOT start hoovering up the dirt already present in my inventory.

It should. ;)

Anyhow, just some observations about an otherwise pretty nifty gimmick. And, I love the name. ;)

**Comment by Entoarox** [comment]
there are plenty of reasons to want to keep those stacks in your inventory, and if you really want to null stacks from an inventory, drag and drop them, problem solved.

**Comment by Altorffer** [comment]
Yes, plenty of reasons to do that indeed, I agree. Plenty of reasons to simply stick /dev/null in some other place, or, hey, remove the item it holds so it will not trigger anymore. Point being -- plenty of reasons, none of which to consider making /dev/null a little more versatile while still offering a lot of other options to maintain its current behaviour.

I think I'll stick to Reliquary's Void Tear instead, does exactly what I proposed while still letting the user be in complete control.

**Comment by Altorffer** [comment]
Plenty of reasons... hmmm, not really plenty, I'd say. Drag and drop them, you say? Yeah, that'll work with a Coin of Fortune... NOT. Point being -- it's not that clear cut really. All I meant to get across and it's sad I have to draw the proverbial schematic for this -- the current behavior of /dev/null is mildly erratic, unexpected, inconsistent. Anyhow... Reliquary's Void Tear will have to do for now.
